routes/web.php

```diff
@@ -379,9 +379,9 @@ function (): void {
                             ->name('manager.jobs.destroy');
 
                         /* Screening Plan Builder */
-                        Route::view(
+                        Route::get(
                             'jobs/{jobPoster}/assessment-plan',
-                            'manager/assessment_plan'
+                            'AssessmentPlanController@show'
                         )
```

Comment:
We should also add `->can('view-assessment-plan', 'jobPoster')` here.

Reply:
`can()` is not a method of Route. Instead added the gate to job policies.

Reply:
My bad. That's supposed to be `->middleware('can:gate-name, 'model')`. I made the same mistake reviewing Chris's PR too.